pkg/apis/componentconfig/types.go

```diff
@@ -160,58 +161,115 @@ const (
 	ProxyModeIPTables  ProxyMode = "iptables"
 )
 
+// SchedulerPolicyConfigMapKey defines the key of the element in the
+// scheduler's policy ConfigMap that contains scheduler's policy config.
+const SchedulerPolicyConfigMapKey string = "policy.cfg"
```

Comment:
pre-existing?  I would have expected an extension like .yaml or something.

Reply:
Yeah, this is left unchanged.

Reply:
> I would have expected an extension like .yaml or something.

Please note that this is not a file name. So, the .cfg is not really an extension that implies the format of the content. It is merely a key to the ConfigMap.